kms/kms_grpc.pb.go

```diff
@@ -20,9 +18,6 @@ import (
 // Requires gRPC-Go v1.32.0 or later.
 const _ = grpc.SupportPackageIsVersion7
 
-// URL of the KMS gRPC endpoint
-var KmsEndpointAddr = os.Getenv("KMS_ENDPOINT_ADDR")
```

Comment:
This was apparently modified manually... If we remove it then it will break many things as nodes won't be able to reach the KMS.

This is not related to your PR, but it would be better if you add a commit with a file under `kms` containing:
```go
package kms

// URL of the KMS gRPC endpoint
var KmsEndpointAddr = os.Getenv("KMS_ENDPOINT_ADDR")
```